vanilla-framework/vanilla-framework
# Refactor to fix the build

## Done
Fix the search icon from crashing the build. Refactored some basic structure of Vanilla.

## QA

- Pull code
- Run `rm -rf build`
- Run `./run build`
- Run `ls -alh build`
- Check the is a build.css and build.min.css files
